GoogleForCreators/web-stories-wp
# Editor: Remove errant 'Edit' option from media menu

## Context

The Call to Action link icon pencil menu displays an option to "Edit" which functions the same as "Upload a file".

**Expected Behaviour**
To have 1 option to "Upload a file".



## Summary

Removed edit link --- noting the same behaviour was happening for shape, image links etc...

https://github.com/GoogleForCreators/web-stories-wp/issues/12555#issuecomment-1294901351


|   before	|   	after |  
|---	|---	|
|   <img width="220" alt="before" src="https://user-images.githubusercontent.com/62242/198825847-3ed3981b-f1ca-46ac-8079-b824ebabfe5a.png">	|   <img width="278" alt="after" src="https://user-images.githubusercontent.com/62242/198825856-bd44ef71-bc9f-4eff-beb1-bca66445a8e7.png">	|  

 








## Relevant Technical Choices



## To-do

Might be worth looking at why clicking the pencil icon multiple times (click (keep menu opne) click pencil again) positions the menu at a different spot - click the menu either ends up off to the left or centered.
 
<img width="214" alt="Screen Shot 2022-10-29 at 6 06 48 AM" src="https://user-images.githubusercontent.com/62242/198826009-39a0dc0d-6aa8-4824-b92e-375553fcbb31.png">





## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add link in the Call to Action section
2. Click pencil on Link Icon
3.  Notice menu option: Upload without edit


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #12555
